src/oss/confluent.sh

```diff
@@ -603,6 +603,11 @@ config_service() {
 
     ${config_command} < "${input_file}" \
         > "${service_dir}/${service}.properties"
+
+    if [[ "${service}" == "connect" ]]; then
```

Comment:
this could use a comment, it's not necessarily obvious why you would do this

Reply:
Good point. Added one.